content/en/docs/collector/deployment/decentralized.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Decentralized
+description:
+  Why and how to send signals to collectors and from there to backends
+weight: 2
+---
+
+The decentralized collector deployment pattern consists of
+applications—[instrumented][instrumentation] with an OpenTelemetry SDK using
+[OpenTelemetry protocol (OTLP)][otlp]—or other collectors (using the OTLP
+exporter) that send telemetry signals to one or more [collectors][collector].
+Each client-side SDK or downstream collector is configured with a collector
+location:
```

Comment:
I found this paragraph difficult to parse. I also found it hard to understand how this pattern differs from the other since you mention several SDKs or collectors sending data to other collectors. Is the difference just that there isn't a load balancer?